dev/com.ibm.ws.microprofile.config.1.2/src/com/ibm/ws/microprofile/config12/converters/ImplicitConverter.java

```diff
@@ -28,117 +33,196 @@
 public class ImplicitConverter extends BuiltInConverter {
```

Comment:
I assume some of these base classes like BuiltinConverter and ImplicitConverter could move to the common bundle at some point?

Reply:
Do you mean `io.openliberty.microprofile.config.internal.common`? That is for the common classes between our original impl and the SmallyRye impl. These changes don't affect the SmallRye impl.

Reply:
I don't remember why that bundle is separate from `io.openliberty.microprofile.config.internal.serverxml`